.node-version

```diff
@@ -1 +1 @@
-18
+20.10.0
```

Comment:
was this intended to be updated?

Reply:
It's intentional, but I realise that I forgot to update the README. I'll add that